scripts/azure-pipelines/linux-build_and_test.sh

```diff
@@ -16,21 +18,40 @@ function or_die () {
     fi
 }
 
-or_die mkdir azure-${TEST_TARGET}-build
-cd azure-${TEST_TARGET}-build
-if [[ "$DO_BUILD" == "yes" ]] ; then
-    or_die cmake -DCMAKE_CXX_COMPILER="${COMPILER}" ${CMAKE_EXTRA_FLAGS} ../src
-    if [[ ${CMAKE_EXTRA_FLAGS} == *COVERAGE* ]] ; then
-      or_die make -j 3
-    else
-      or_die make -j 3 VERBOSE=1
-    fi
-    if [[ "${DO_TEST}" == "yes" ]] ; then
-      or_die ctest -T test --output-on-failure -V
-    fi
-    if [[ "${DO_MEMCHECK}" == "yes" ]] ; then
-      or_die ctest -T memcheck
-    fi
-fi
+or_die cd axom
+git submodule init 
+git submodule update 
+
+echo HOST_CONFIG
+echo $HOST_CONFIG
+
+./config-build.py -hc /home/axom/axom/host-configs/docker/${HOST_CONFIG}.cmake -DENABLE_GTEST_DEATH_TESTS=ON
+
+or_die cd build-$HOST_CONFIG-debug
+or_die make -j 3
+make CTEST_OUTPUT_ON_FAILURE=1 test ARGS='-T Test -VV -j8'
+
+
+find ./axom/sidre -type d -exec chmod 755 {} \;
+find ./axom/sidre -type f -exec chmod 644 {} \;
```

Comment:
@keithhealy -- are these ``chmod``s necessary? Or was it part of your testing?

Reply:
Sorry I commented in the wrong place.  The chmod commands were necessary.  The find "*.xml" is not and I'll remove it.